docker-compose.yaml

```diff
@@ -21,9 +22,20 @@ services:
     restart: always
     # This will check if your certificate is up for renewal every 12 hours as recommended by Let’s Encrypt
     entrypoint: "/bin/sh -c 'trap exit TERM; while :; do certbot renew; sleep 12h & wait $${!}; done;'"
- 
+
   registry:
     restart: always
     image: registry:2
     volumes:
+      - /etc/letsencrypt:/etc/letsencrypt
+      - ./registry-config.yaml:/etc/docker/registry/config.yml
```

Comment:
Let's move this `registry-config.yaml` file to `config/registry/config.yml` so that it more closely mirrors where it will go inside the container.